distribution/test/Directory.Build.props

```diff
@@ -11,7 +11,15 @@
   </PropertyGroup>
 
   <ItemGroup Label="Test Analyzers">
-    <PackageReference Include="NSubstitute.Analyzers.CSharp" Version="1.0.14" PrivateAssets="All" />
+    <PackageReference Include="NSubstitute.Analyzers.CSharp" Version="1.0.15" PrivateAssets="All" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+  <ItemGroup>
```

Comment:
Consider if we want Atc.test to be included by default as well.

Reply:
Did think about that. Are we using that in all new test projects?

Reply:
If we are, we might want to include the package reference for it by default in our Directory.Build.props (perhaps also for NSubstitute and Autofixture).

Reply:
> Did think about that. Are we using that in all new test projects?

I do and I think we generally do.

Reply:
just experimented a bit. It seems that if you add a using in the csproj whose package is not referenced, nothing breaks.